unified-scheduler-pool/src/lib.rs

```diff
@@ -648,6 +671,10 @@ where
     }
 
     fn is_trashed(&self) -> bool {
```

Comment:
so want to make a clarification here in the review, doesn't necessarily need a code comment.

This is **only** called after the scheduler is terminated due to abort or successful completion of the slot, is that right?
I want to make sure that the limits from `is_overgrown` will not cause us to skip a slot if such a large one comes through.

Reply:
> This is **only** called after the scheduler is terminated due to abort or successful completion of the slot, is that right?

yes. it's only called by `::return_to_pool()`.

> I want to make sure that the limits from `is_overgrown` will not cause us to skip a slot if such a large one comes through.

yes again.